### PR TITLE
Add logging and a console to show the logs

### PR DESCRIPTION
**Pull request by mee4895**

### CODE REVIEW

File: ICGE-Ui/src/main/java/de/unistuttgart/informatik/fius/icge/ui/internal/ConsoleBufferedOutputStream.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+ * text into a {@link JTextArea}
+ *
+ * @author Tobias Wältken
+ * @version 1.0
+ */
+public class ConsoleBufferedOutputStream extends OutputStream {
+
+    private JTextArea textArea;
+
+    //TODO add actual buffer to avoid overflowing the textarea and cause lag
```
Comment (neumantm) [inline ICGE-Ui/src/main/java/de/unistuttgart/informatik/fius/icge/ui/internal/ConsoleBufferedOutputStream.java]: Should this not be done in this PR?

Comment (mee4895) [reply ICGE-Ui/src/main/java/de/unistuttgart/informatik/fius/icge/ui/internal/ConsoleBufferedOutputStream.java]: Yes it should but I wanted to postpone that cause it is not necessary and we have more pressing things that should be done, thus a TODO comment. Hope that isn't a large issue.

**Comment by mee4895** [comment]
> I will also run format and cleanup over everything once you are done.

Can you tell me what format things you don't like so I can change that in the future

File: ICGE-Ui/src/main/java/de/unistuttgart/informatik/fius/icge/ui/internal/ConsoleBufferedOutputStream.java (hunk 1 above)
Comment (neumantm) [reply ICGE-Ui/src/main/java/de/unistuttgart/informatik/fius/icge/ui/internal/ConsoleBufferedOutputStream.java]: ok

**Comment by neumantm** [comment]
> > I will also run format and cleanup over everything once you are done.
> 
> Can you tell me what format things you don't like so I can change that in the future

See my commt

**Pull request merged**